nixos/modules/services/web-apps/flarum.nix

```diff
@@ -194,8 +201,7 @@ in {
         cp -f ${cfg.package}/share/php/flarum/{extend.php,site.php,flarum} .
         ln -sf ${cfg.package}/share/php/flarum/vendor .
         ln -sf ${cfg.package}/share/php/flarum/public/index.php public/
-        chmod a+x . public
-        chmod +x site.php extend.php flarum
+        chmod 775 -R ${cfg.stateDir}
```

Comment:
Why does it differ from `homeMode = "755";`?

Reply:
Just to ensure that the files and directory inside the stateDir will have the 755 permission. As stated in: https://docs.flarum.org/install/#folder-ownership.

Reply:
Applied `homeMode = "755"` only.